packages/cookie_store/packages/cookie_store_conformance_tests/pubspec.yaml

```diff
@@ -3,19 +3,17 @@ description: >-
   A library that tests whether implementations of package:cookie_store's `CookieStore`
   class behave as expected.
 publish_to: none
+resolution: workspace
+
 environment:
-  sdk: ^3.0.0
+  sdk: ^3.5.0
 
 dependencies:
   cookie_store:
-    git:
-      url: https://github.com/nextcloud/neon
-      path: packages/cookie_store
+    path: ../..
   meta: ^1.0.0
   test: ^1.21.2
 
 dev_dependencies:
   neon_lints:
-    git:
-      url: https://github.com/nextcloud/neon
-      path: packages/neon_lints
+    path: ../../../neon_lints
```

Comment:
We should have it as a dev_dependency of the workspace package and just use a global `analysis_options.yaml`.

Every package may still alter the rule set but this would make our lints the default (also for newly added packages).

Reply:
I didn't think about this, but yes it makes sense. How are we going to handle custom_lint though? We need to disable some of the rules for specific packages?
OTOH this brings up the point of removing custom_lint entirely, given that it mostly causes headaches and hasn't caught so many problems so far.